src/Shipment/TaxIdentificationNumbers/TaxIdentificationNumbers.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelCom\Integration\Shipment\TaxIdentificationNumbers;
+
+use ArrayObject;
+use function array_map;
+
+class TaxIdentificationNumbers extends ArrayObject
```

Comment:
Lets name this one `TaxIdentificationNumberCollection` instead because `TaxIdentificationNumbers` and `TaxIdentificationNumber` are too close in their spelling and for me they are easy to mistake. Furthermore, `items` is already called `ItemCollection`
```suggestion
class TaxIdentificationNumberCollection extends ArrayObject
```

Reply:
Agreed

Reply:
Done